docs/CHANGELOG.rst

```diff
@@ -138,6 +138,7 @@ Fixed
   ``resolwebio/rnaseq`` Docker image
 * Fix handling of NA values in Differential expression results table. NA values
   were incorrectly replaced with value 0 instead of 1
+* Fix `archive-samples` to use sample names for headers when merge expressions
```

Comment:
Use double backticks in the change log. _Merging_ expressions would sound better.

While you're at it, please also add a change log entry for [GOTEA](https://github.com/genialis/resolwe-bio/pull/506/files) and fix backticks [here](https://github.com/genialis/resolwe-bio/pull/510/files#diff-7c2bbd961025070ab687adf18fc72ba3R122).